Zufallsgedicht/Main.ts

```diff
@@ -23,12 +23,12 @@ for (let i: number = 0; i < subjects.length; i++) {
 
 appendToHTML(output);
 
-function shuffle(array: any[]) {
+function shuffle(array: any[]) {  //ich weiß nicht, ob das hier zutrifft, aber "any" sollte nicht benutzt werden
```

Comment:
Da die Funktion nicht shuffeStringArray oderso heißt, geht man davon aus, dass diese alles shuffelt egal welcher datentyp der Array hat. Deshalb ist hier any sinnvoller in meinen Augen